models/icatcher-semireal/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "download": {
+    "file": "https://civitai.com/api/download/models/556730",
```

Comment:
Thanks for submission. A few things: first make sure this is accessible without login (I think so, just want to confirm).

Reply:
all my models should already be accessible without login.